PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -140,6 +140,7 @@
 
 _pp_on_AA_extraCommands = [
     'keep patPackedCandidates_hiPixelTracks_*_*',
+    'keep patPackedCandidates_packedPFCandidatesCleaned_*_*',
```

Comment:
```suggestion
    'keep patPackedCandidates_packedPFCandidatesRemoved_*_*',
```

Reply:
just to be sure, is this the extras that were otherwise removed from the default `packedPFCandidates`? I just wanted to make sure that the names are somewhat self-descriptive

Reply:
Yes, these are the packedPFCandidates that are no longer in the default collection.  Regarding the commit suggestion, I missed this one when updating the naming convention during a previous round of review.  Fixing now.